libraries/AP_Scripting/applets/pelco_d_antennatracker.lua

```diff
@@ -0,0 +1,112 @@
+--[[
+   Pelco-D control implementation for antennatracker.
+   Implemented by using knowledge from the excellent python implementation in https://gist.github.com/jn0/cc5c78f4a0f447a6fb2e45a5d9efa13d.
+--]]
+
+---@diagnostic disable: need-check-nil
```

Comment:
This is cheating. The scripting checks are still quite new, so we have a bunch of these in the code base for pre-exiting scripts. I would like to not add any more. 

What is the error it gives?

Reply:
The assert should have fixed that?

Reply:
Looks like were good.